integration/container/inspect_test.go

```diff
@@ -68,3 +69,28 @@ func TestInspectAnnotations(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Check(t, is.DeepEqual(inspect.HostConfig.Annotations, annotations))
 }
+
+func TestNetworkAliasesAreEmpty(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType == "windows")
+
+	ctx := setupTest(t)
+	apiClient := request.NewAPIClient(t)
```

Comment:
double-check; does this test depend on API version, or is this for all versions?